dockerfiles/broker.dockerfile

```diff
@@ -45,8 +53,9 @@ RUN \
     --mount=type=cache,target=/root/.cache/sccache/,id=bndlss_broker_sc \
     source ./sccache-config.sh ${S3_CACHE_PREFIX} && \
     cargo build --release --bin broker && \
-    cp /src/target/release/broker /src/broker && \
-    sccache --show-stats
+    cp /src/target/release/broker /src/broker
+
+RUN sccache --show-stats
```

Comment:
why? the less `run` the faster in theory no?

Reply:
Forgot to remove after debugging, moving back